### PR TITLE
Fix botUID in chatbot

### PR DESCRIPTION
**Pull request by THUzxj**
Hi,

Thanks for your great project. I am working on deploying Tinode and developing chatbots. The `botUID` in the current chatbot example does not work because it is always `None`. So this PR fixes it.

By the way, I can help build a unified Python API for both chatbot and tn-cli, based on the code of tn-cli. What is your opinion on this? Thank you!

### CODE REVIEW

File: chatbot/python/chatbot.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def client_message_loop(stream):
                 exec_future(msg.ctrl.id, msg.ctrl.code, msg.ctrl.text, msg.ctrl.params)
 
             elif msg.HasField("data"):
-                # log("message from:", msg.data.from_user_id)
+                # log("message from:", msg.data.from_user_id, "botUID:", botUID)
```
Comment (or-else) [inline chatbot/python/chatbot.py:259]: I don't think this change is needed.

File: chatbot/python/chatbot.py [hunk 2]
```diff
@@ -309,6 +310,8 @@ def on_login(cookie_file_name, params):
 
     if 'user' in params:
         botUID = params['user'].decode("ascii")
+        if botUID[0] == '"':
```
Comment (or-else) [inline chatbot/python/chatbot.py:313]: Why is this needed?

**Comment by or-else** [comment]
Thank you for your PR. Please use `devel` branch.

File: chatbot/python/chatbot.py (hunk 1 above)
Comment (THUzxj) [reply chatbot/python/chatbot.py:259]: Yes, it is just for debugging.

File: chatbot/python/chatbot.py (hunk 2 above)
Comment (THUzxj) [reply chatbot/python/chatbot.py:313]: `params['user'].decode("ascii")` returns the string containing quotation marks in the front and end, like '"usrxxxxx"'. I am not sure it will always happen in any environments.

File: chatbot/python/chatbot.py (hunk 1 above)
Comment (or-else) [reply chatbot/python/chatbot.py:259]: Yes, I understand, but you don't need to change an existing line. Or you can remove it alltogether.

File: chatbot/python/chatbot.py (hunk 2 above)
Comment (or-else) [reply chatbot/python/chatbot.py:313]: It's always quoted because values `params` dictionary are bytes. So, just always unquote it:

```python
# Values params are bytes, so a string is always quoted. Unquote it.
botUID = params['user'].decode("ascii")[1:-1]
```

File: chatbot/python/chatbot.py (hunk 1 above)
Comment (THUzxj) [reply chatbot/python/chatbot.py]: Removed the change.

File: chatbot/python/chatbot.py (hunk 2 above)
Comment (THUzxj) [reply chatbot/python/chatbot.py]: Thanks, got it and fixed.

**Review by or-else — APPROVED** [review]
